Simplenote/AccountDeletionController.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+@objc
+class AccountDeletionController: NSObject {
+    private var accountDeletionRequestDate: Date?
+
+    var deletionTokenHasExpired: Bool {
```

Comment:
Nipticky: how about if we rename this to `mustReauthenticateSimperium` and we invert the conditions? (it'd be way more clear what should happen, etc!)